config.json

```
@@ -1143,6 +1143,62 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
-  "tags": []
+  "key_features": [
+    {
+      "title": "Evolving",
+      "content": "Typescript is a modern and constantly evolving language supported by Microsoft."
+    },
+    {
+      "icon": "features-cross-platform",
+      "title": "Widely used",
+      "content": "Typescript is used by many frontend frameworks, and can be used in Javascript based ones as well."
+    },
+    {
+      "icon": "features-strongly-typed",
+      "title": "Strongly typed",
+      "content": "Typescript flexible typing system lets you describe what to expect of your code easily."
```

```suggestion
      "content": "Typescript flexible typing system lets you describe what to expect, supporting gradual adoption."
```

I think this will be nice to mention as people worry about this.

---

True. Kotlin heavily emphasizes on this for example, and is true and important for TS as well

---

I adore Kotlin too ;)